bom/pom.xml

```diff
@@ -217,7 +217,7 @@ THE SOFTWARE.
       <dependency>
         <groupId>org.jenkins-ci</groupId>
         <artifactId>bytecode-compatibility-transformer</artifactId>
-        <version>2.0-beta-2</version>
+        <version>2.0-SNAPSHOT</version>
```

Comment:
ummm, @oleg-nenashev feel like releasing a GA version?

Reply:
With https://github.com/jenkinsci/bytecode-compatibility-transformer/pull/19 - sure
I updated the BCT permissions so that all core team members can access it